packages/db/src/prisma.ts

```diff
@@ -0,0 +1,18 @@
+import { PrismaClient } from '@prisma/client';
+
+const prismaClientSingleton = () => {
```

Comment:
any reason for this in this change?

Reply:
learnt about turborepo and prisma now
reverting this